tests/pageObjects/LoginPage.js

```diff
@@ -0,0 +1,22 @@
+class LoginPage {
+constructor() {
+    this.loginBtnLocator = "(//input[@name='commit'])[1]";
+    this.emailFieldLocator = "#spree_user_email"
+    this.passwordFieldLocator = "#spree_user_password"
+    this.errorLocator = ".alert-danger"
+}
+ async navigate() {
+    await page.goto("http://localhost:3000/admin/login");
+ }
+
+ async loginInputFields(email, password ){
+    await page.fill(this.emailFieldLocator, email);
+    await page.fill(this.passwordFieldLocator,password);
+}
+
+async clickLoginBtn() {
+    await page.click(this.loginBtnLocator); 
+}
```

Comment:
Combine these functions into one
```suggestion
 async login(email, password ){}
```